SPECS/nexus-oss.spec

```diff
@@ -12,14 +12,15 @@
 Summary: Nexus manages software “artifacts” required for development, deployment, and provisioning.
 Name: nexus
 Version: 2.14.5.02
-Release: 1%{?dist}
+Release: 2%{?dist}
 # This is a hack, since Nexus versions are N.N.N-NN, we cannot use hyphen inside Version tag
 # and we need to adapt to Fedora/SUSE guidelines
 %define nversion %(echo %{version}|sed -r 's/(.*)\\./\\1-/')
 License: AGPL
 Group: unknown
 URL: http://nexus.sonatype.org/
 Source0: http://www.sonatype.org/downloads/%{name}-%{nversion}-bundle.tar.gz
+Source1: %{name}.service
 BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root
 Requires(pre): /usr/sbin/useradd, /usr/bin/getent
 # As soon as we implement a systemd service, this ugly require can be removed
```

Comment:
Will deleted?

Reply:
Yes, this needs to be removed for both nexus and nexus3

Reply:
Deleted